[DOC] Update Management navigation interface in dashboards for UX revisions and add index management tutorial

vagimeli opened the pull request:
### Description
Updates UI terminology and interface changes; adds tutorial 

### Issues Resolved
https://github.com/opensearch-project/documentation-website/issues/4364


### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

natebower commented on _dashboards/dev-tools/run-queries.md:
```diff
@@ -1,33 +1,23 @@
 ---
 layout: default
-title: Running queries in the Dev Tools console
+title: Running queries in the Dev Tools Console
```
It looks like we actually don't capitalize "console" above.

vagimeli replied:
Changed to lowercase across docs

natebower commented on _dashboards/dev-tools/run-queries.md:
```diff
@@ -1,33 +1,23 @@
 ---
 layout: default
-title: Running queries in the Dev Tools console
+title: Running queries in the Dev Tools Console
```
```suggestion
title: Running queries in the Dev Tools console
```

natebower commented on _dashboards/dev-tools/run-queries.md:
```diff
@@ -1,33 +1,23 @@
 ---
 
-# Running queries in the Dev Tools console
+# Running queries in the Dev Tools Console
```
```suggestion
# Running queries in the Dev Tools console
```

natebower commented on _dashboards/dev-tools/run-queries.md:
```diff
@@ -1,33 +1,23 @@
-You can open the console from any other page by navigating to the main menu and selecting **Management** > **Dev Tools**.
-
-<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/dev-tools-left.png" width=200 alt="Dev Tools Console from all pages">
+Use the Dev Tools Console to send queries to OpenSearch. To access the Dev Tools Console, select **Dev Tools** under the **Management** menu on the OpenSearch Dashboards home page. 
```
```suggestion
Use the Dev Tools console to send queries to OpenSearch. To access the Dev Tools Console, select **Dev Tools** under the **Management** menu on the OpenSearch Dashboards home page. 
```

natebower commented on _dashboards/discover/time-filter.md:
```diff
@@ -11,7 +11,7 @@ redirect_from:
 You can change the time range to display dashboard data over minutes, hours, days, weeks, months, or years.
 
-The default time range is **Last 15 minutes**. You can change the time range at the dashboard level or under **Stack Management > Advanced Settings > Time filter defaults**. 
+The default time range is **Last 15 minutes**. You can change the time range at the dashboard level or under **Dashboards Management** > **Advanced Settings** > **Time filter defaults**. 
```
As with this one. I'm not 100% on that, though, so just confirm.

natebower commented on _dashboards/im-dashboards/index.md:
```diff
@@ -11,6 +11,22 @@ redirect_from:
+
+**Step 2: View indexes** In the Indices interface you will see a list of existing indexes in your OpenSearch cluster. The list provides information such as index name, health state, document count, index size, and other relevant details.  
+
+**Step 3: Create an index** To create a new index, select the **Create index** button in the upper right corner. You will be prompted to enter the index name and configure the index settings, such as number of shards and replicas. Fill in the required information and select **Create** to create the index.
```
```suggestion
**Step 3: Create an index** To create a new index, select the **Create index** button in the upper-right corner. You will be prompted to enter the index name and configure the index settings, such as number of shards and replicas. Fill in the required information and select **Create** to create the index.
```

natebower commented on _dashboards/im-dashboards/index.md:
```diff
@@ -11,6 +11,22 @@ redirect_from:
+
+**Step 3: Create an index** To create a new index, select the **Create index** button in the upper right corner. You will be prompted to enter the index name and configure the index settings, such as number of shards and replicas. Fill in the required information and select **Create** to create the index.
+
+**Step 4: Delete an index** To delete an index, locate the index and select the checkbox next to it. Then, select the **Actions** button and choose **Delete** from the dropdown list. Use caution when deleting indexes, as this action is irreversible. 
```
```suggestion
**Step 4: Delete an index** To delete an index, locate the index and select the checkbox next to it. Then select the **Actions** button and choose **Delete** from the dropdown list. Use caution when deleting indexes because this action is irreversible. 
```

natebower commented on _dashboards/im-dashboards/index.md:
```diff
@@ -11,6 +11,22 @@ redirect_from:
+
+**Step 4: Delete an index** To delete an index, locate the index and select the checkbox next to it. Then, select the **Actions** button and choose **Delete** from the dropdown list. Use caution when deleting indexes, as this action is irreversible. 
+
+**Step 5: Modify an index** To modify the settings of an existing index, locate the index in the list and select its name. This takes you to the index details page. Here, you can update settings such as the numbers of shards, replicas, and other advanced configurations. After making the desired changes, select **Save**.
```
```suggestion
**Step 5: Modify an index** To modify the settings of an existing index, locate the index in the list and select its name. This takes you to the index details page. Here you can update settings such as the numbers of shards, replicas, and other advanced configurations. After making the desired changes, select **Save**.
```

natebower commented on _dashboards/im-dashboards/index.md:
```diff
@@ -11,6 +11,22 @@ redirect_from:
+
+**Step 5: Modify an index** To modify the settings of an existing index, locate the index in the list and select its name. This takes you to the index details page. Here, you can update settings such as the numbers of shards, replicas, and other advanced configurations. After making the desired changes, select **Save**.
+
+**Step 7: Refresh indexes** To refresh an index, locate the index and select the checkbox next to it. Then, select the **Actions** button and choose **Refresh** from the dropdown list.
```
```suggestion
**Step 7: Refresh indexes** To refresh an index, locate the index and select the checkbox next to it. Then select the **Actions** button and choose **Refresh** from the dropdown list.
```

natebower commented on _dashboards/im-dashboards/index.md:
```diff
@@ -11,6 +11,22 @@ redirect_from:
+
+**Step 7: Refresh indexes** To refresh an index, locate the index and select the checkbox next to it. Then, select the **Actions** button and choose **Refresh** from the dropdown list.
+
+**Step 8: Filter and search indexes** If you have a large number of indexes and want to filter or search for specific indexes, you can use the search bar located above the list of indexes. Enter the relevant keywords or filters to narrow down the displayed indexes.
```
```suggestion
**Step 8: Filter and search indexes** If you have a large number of indexes and want to filter or search for specific indexes, you can use the search bar located above the list of indexes. Enter the relevant keywords or filters to narrow the list of indexes.
```

natebower commented on _dashboards/im-dashboards/index.md:
```diff
@@ -11,6 +11,22 @@ redirect_from:
+
+**Step 8: Filter and search indexes** If you have a large number of indexes and want to filter or search for specific indexes, you can use the search bar located above the list of indexes. Enter the relevant keywords or filters to narrow down the displayed indexes.
+
+**Step 9: Additional operations** Index Management provides additional functionalities such as creating index patterns, managing lifecycle policies, and configuring index templates. These options are available in their respective sections of the Index Management interface. You can explore them based on your requirements. 
```
The last sentence reads as potentially superfluous.

natebower commented on _dashboards/im-dashboards/notifications.md:
```diff
@@ -20,22 +20,22 @@ On the **Notification settings** page, you can configure the default notificatio
 ## Configuring notification settings for an individual operation
 
-You can view default notification settings when you perform an indexing operation. You can also set up additional notifications. For example, if you want to configure an additional notification for a reindex operation, perform the following steps:
+You can view default notification settings when you perform an indexing operation, along with set up additional notifications. For example, if you want to configure an additional notification for a reindex operation, perform the following steps:
```
```suggestion
You can view default notification settings when you perform an indexing operation as well as set up additional notifications. For example, if you want to configure an additional notification for a reindex operation, perform the following steps:
```

natebower commented on _dashboards/im-dashboards/notifications.md:
```diff
@@ -54,7 +54,7 @@ You can view default notification settings when you perform an indexing operatio
 1. Select a channel from the **Notification channels** dropdown list. If you want to configure a new notification channel, select **Manage channels**.
     
-    To configure a new notification channel, ensure the `dashboards-notification` plugin is enabled in OpenSearch Dashboards. 
+    To configure a new notification channel, confirm the `dashboards-notification` plugin is enabled in OpenSearch Dashboards. 
```
```suggestion
    To configure a new notification channel, confirm that the `dashboards-notification` plugin is enabled in OpenSearch Dashboards. 
```

natebower commented on _dashboards/sm-dashboards.md:
```diff
@@ -6,11 +6,9 @@ redirect_from:
 ---
 
-# Snapshot management
+# Snapshot management in Dashboards
```
```suggestion
# Snapshot Management in Dashboards
```

natebower commented on _dashboards/sm-dashboards.md:
```diff
@@ -6,11 +6,9 @@ redirect_from:
-You can set up Snapshot Management (SM) in OpenSearch Dashboards.
-
-[Snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/index/) are backups of a cluster’s indexes and state. The state includes cluster settings, node information, index metadata (mappings, settings, templates), and shard allocation.
+[Snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/index/) are backups of a cluster’s indexes and state. The state includes cluster settings, node information, index metadata (mappings, settings, templates), and shard allocation. The Snapshot Management interface in OpenSearch Dashboards provides a unified solution for taking and restoring snapshots.
```
```suggestion
[Snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/index/) are backups of a cluster’s indexes and state. The state includes cluster settings, node information, index metadata (mappings, settings, templates), and shard allocation. The Snapshot Management (SM) interface in OpenSearch Dashboards provides a unified solution for taking and restoring snapshots.
```

natebower commented on _dashboards/sm-dashboards.md:
```suggestion
Before you create an SM policy, set up a repository for snapshots. 
```

natebower commented on _dashboards/sm-dashboards.md:
````diff
@@ -46,6 +40,9 @@ Before you create an SM policy, you need to set up a repository for snapshots.
     ```
 1. Choose the **Add** button.
 
+If you need to automate snapshots creation, you can use a snapshot policy.
````
```suggestion
If you need to automate snapshot creation, you can use a snapshot policy.
```

natebower reviewed (approved):
@vagimeli Approved with noted changes and comments. Let me know if you have any questions. Thanks!

vagimeli commented:
> @vagimeli Approved with noted changes and comments. Let me know if you have any questions. Thanks!

@natebower I accepted your changes and addressed comments. The proper format is "Select X > Y >Z".

vagimeli commented on _dashboards/discover/multi-data-sources.md:
```diff
@@ -32,7 +32,7 @@ A data source connection specifies the parameters needed to connect to a data so
 
 1. Go to [`http://localhost:5601`](http://localhost:5601/) and log in with the username `admin` and password `admin`. If you’re running the Security plugin, go to [`https://localhost:5601`](https://localhost:5601/).
-2. From the OpenSearch Dashboards main menu, select **Stack Management**, **Data Sources**, and then **Create data source connection**.
+2. From the OpenSearch Dashboards main menu, select **Dashboards Management**, **Data Sources**, and then **Create data source connection**.
```
> is proper usage.
